Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -84,6 +86,7 @@ namespace AZ
 
             struct LodConfiguration
             {
+                AssetQuality m_lodBias = 0;
```

Comment:
Instead of putting the bias in the Culling, I think it would be better to put it in ModelDataInstance::BuildCullable to keep the data and logic in culling as minimal as we can (it's already got a lot). Then any bias can be applied once whenever the bias/lod configuration changes for an object, instead of once every frame for every view for every object. m_lodBias doesn't need to be in the configuration at all. 

You can adjust the m_screenCoverageMin and m_screenCoverageMax in LodData::Lod, in ModelDataInstance::BuildCullable, and adjust the LodConfiguration::m_lodOverride in the MeshComponentController before calling SetMeshLodConfiguration (you would have to do it everywhere SetMeshLodConfiguration is called, but you can store the bias in a member variable of the MeshComponent so you don't have query the tag system every time).

Also, you'd need to factor out the screenCoverageMin/Max calculations into a function, and call that function in ModelDataInstance::SetMeshLodConfiguration (which I think needs to be done anyways, it looks like that might be a bug where changing the configuration on the component doesn't update the min/max values, unless something else is forcing the cullable to rebuild). It would also allow you to eliminate the issue I commented on in Culling.cpp, where it looks the object would stop being rendered entirely (unless that's the desired effect). You could either adjust the step size so the remaining lods are still evenly distributed across screen size percentages, or just make the final lod occupy a larger range than the rest.

Then Culling wouldn't need to be modified at all, because the bias would be baked in.

Reply:
That sounds great, is there a way to ensure that BuildCullable will be called again (to ensure an updated lod bias/asset quality will be taken into account)?

Reply:
I implemented it